src/main.rs

```diff
@@ -12,19 +13,19 @@ enum Command {
     Login(login::ContestInfo),
 
     #[structopt(name = "submit", about = "Submits FILE to given PROBLEM.")]
-    Submit(submit::Submission),
+    Submit(client::Submission),
 }
 
 #[tokio::main]
 async fn main() -> Result<()> {
     match Command::from_args() {
         Command::Login(contest_info) => {
-            login::read_login(&contest_info)
+            ejudge::read_login(&contest_info)
```

Comment:
Why by default login using ejudge flow is executed? 
It must be either subcommand or one of the arguments [probably, guessed from URL].
```sh
$ cpt login ejudge --url CONTEST_URL --contest_id CONTEST_ID
$ cpt login --url CONTEST_URL --contest_id CONTEST_ID [--client ejudge]

Reply:
Let's use subcommands for now. Seems a bit more annoying, but much easier to scale. Let's reconsider it after more judge systems already adopted.